editor/src/utils/react-performance.ts

```diff
@@ -209,6 +209,86 @@ function failSafeMemoEqualityFunction(componentDisplayName: string, severity: 's
   }
 }
 
+function keepDeepReferenceEqualityOfObject(
```

Comment:
https://github.com/epoberezkin/fast-deep-equal/ has been updated to support ES6 Maps. since this code here is taken from them, I think we should adopt the current version instead of deviating from it, AFAIK they are still the fastest deep equals out there

Reply:
For the benefit of anyone else reading this - we discussed this on the phone and Balazs is going to pick this up. I have removed the change from this PR